x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -43,9 +49,13 @@ export function calculateDatafeedFrequencyDefaultSeconds(bucketSpanSeconds: numb
   return freq;
 }
 
+export function isTimeSeriesViewJob(job: CombinedJob): boolean {
```

Comment:
This function needs a comment to clarify how it is different to `isTimeSeriesViewableJob` and when to use one rather than the other.

Reply:
Updated here https://github.com/elastic/kibana/pull/84594/commits/b47459d369ba5485d798e7a976bed1b25c4f5c94